ui/Screen/Onboarding/Success.svelte

```diff
@@ -48,8 +48,8 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your <span class="typo-text-bold">Radicle ID</span>. Click to copy
-      it and share it with others so that they can find you.
+      This is your Radicle ID — it's unique to you!<br /> Click to copy it and share
```

Comment:
```suggestion
      This is your Radicle ID — it uniquely identifies you in the Radicle Network!<br /> Click to copy it and share
```

"it's unique do you" sounds like it's unique within my ownership, not that it is unique in the network. Feel free to reformulate in any other way :)

Reply:
I took this directly from the Figma mockup for beta 1. There was already a discussion on the exact wording, I merely copied it here. If you want to suggest changes to the wording, it probably makes sense to do it [there](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA?node-id=4147:7852#39220123). \cc @abbey-titcomb